### ISSUE TITLE
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime

### ISSUE THREAD

**Issue by YAIsaienkov** [tensorflow/tensorflow#34868]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Ubuntu 16.04
- TensorFlow installed from (source or binary):
pip install tensorflow
- TensorFlow version (or github SHA if from source):
2.1.0rc0

**Provide the text output from tflite_convert**

```
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: ADD, FULLY_CONNECTED, LOGISTIC, MUL, RESHAPE, STRIDED_SLICE. Here is a list of operators for which you will need custom implementations: TensorListFromTensor, TensorListReserve, TensorListStack, While.
```

Also, please include a link to a GraphDef or the model if possible.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by jvishnuvardhan** [comment]
@YAIsaienkov Can you please share standalone code to reproduce the issue? Thanks!

**Comment by YAIsaienkov** [comment]
`    model = load_model(hdf_filename)
    model.compile(loss='binary_crossentropy', optimizer='adam')
    converter = tf.lite.TFLiteConverter.from_keras_model(model)

    tflite_model = converter.convert()

    with open(tflite_filename, 'wb') as f:
        f.write(tflite_model)`
`    model = Sequential()
    model.add(LSTM(16, input_shape=input_shape))
    model.add(Dropout(.2))
    model.add(BatchNormalization())
    model.add(Dense(16))
    model.add(Dense(1, activation='sigmoid'))
    # print(model.summary())
`

**Comment by jvishnuvardhan** [comment]
@YAIsaienkov I created a colab with your code and ran it using new experimental converter as follows
`converter.experimental_new_converter = True` 

Everything worked without an issue. I am attaching the [gist](https://colab.sandbox.google.com/gist/jvishnuvardhan/469e50fe41de23a93c4ae62de67b6770/untitled695.ipynb) for your reference. Thanks!

Please close the issue if this resolved your issue. Thanks!

**Comment by YAIsaienkov** [comment]
Thank's!
But I have one more question:
Before I convert model I could feed to the model shapes (n, 20, 8), but now only (1, 20, 8). Should it be like that?

**Comment by gargn** [comment]
If your batch size is always intended to be the same non-1 value, we recommend setting the batch size of the input tensor in the TensorFlow model (e.g. instead of (n, 20, 8) have the model shape be (10, 20, 8)).

However, if that's not the case, then we currently don't support dynamic shapes. There are a few other open issues about adding dynamic shape support: https://github.com/tensorflow/tensorflow/issues/24607, https://github.com/tensorflow/tensorflow/issues/33711. It's an issue we are aware of and intend to resolve.

One solution that works for some users is to call `resize_input_tensor` on the Interpreter with new input shape having the desired batch size. That allows you to change the batch size during inference. However, this is just a work around and doesn't work for most models (because of the way the model is constructed).

**Comment by YAIsaienkov** [comment]
Thank you all for the help!
